README.md

```diff
@@ -30,6 +30,17 @@ Please visit the [documentation].
 - [pyevm] (in progress)
 - [pyethereum] (abandoned)
 
+## Internal
```

Comment:
We usually put this into a different file. In any case it seems to be belong at the end of this file.

Reply:
Yeah, but https://github.com/RichardLitt/standard-readme/blob/master/spec.md#extra-sections.